src/lib/comms/include/sol-network.h

```diff
@@ -73,6 +73,9 @@ extern "C" {
  */
 #define SOL_INET_ADDR_STRLEN 48
 
+#ifndef AF_BT_IOTIVITY
+    #define AF_BT_IOTIVITY 31
```

Comment:
I still don't think it is right. What is the purpose of mixing Iotivity knowledge in sol-socket level?